Bindings/bindings.json

```diff
@@ -383,6 +383,9 @@
                         "capability": "blob"
                     }
                 }
+            ],
+            "filters": [
+                "showHiddenBinding"                
```

Comment:
Actually we want to release those. Please do not hide those
On May 3, 2016 07:58, "Naren Soni" <notifications@github.com> wrote:

> In Bindings/bindings.json
> <https://github.com/Azure/azure-webjobs-sdk-templates/pull/174#discussion_r61895827>
> :
>
> > @@ -383,6 +383,9 @@
> >                          "capability": "blob"
> >                      }
> >                  }
> > +            ],
> > +            "filters": [
> > +                "showHiddenBinding"
>
> showHiddenBinding is the query string key you need to add show ApiHub
> Binding. It is hidden by default when the filter setting is present. My
> understanding was that we wanted hide API Hub File bindings.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Azure/azure-webjobs-sdk-templates/pull/174/files/1aab6f57efd1dba33bcf47bb6253112c92ef1045#r61895827>
>